tests/dcps_tests.lst

```diff
@@ -140,8 +140,8 @@ tests/DCPS/Presentation/run_test.pl: !DCPS_MIN !DDS_NO_OBJECT_MODEL_PROFILE !DDS
 tests/DCPS/ReaderDataLifecycle/run_test.pl: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE
 tests/DCPS/ReaderDataLifecycle/run_test.pl rtps: !DCPS_MIN RTPS !DDS_NO_OWNERSHIP_PROFILE
 tests/DCPS/Reconnect/run_test.pl: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE
-tests/DCPS/Reconnect/run_test.pl restart_sub: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !GH_ACTIONS_W16
-tests/DCPS/Reconnect/run_test.pl restart_pub: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !GH_ACTIONS_W16
+tests/DCPS/Reconnect/run_test.pl restart_sub: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !GH_ACTIONS_W22
+tests/DCPS/Reconnect/run_test.pl restart_pub: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !GH_ACTIONS_W22
```

Comment:
Have you tried running without having the `!GH_ACTIONS_*` in the list?  Do these tests actually fail on the new platform / configurations?

Reply:
No, I haven't tried running these tests on Windows 2022. I wasn't sure why we had `!GH_ACTIONS_W16` for Windows 2016 and so just mechanically updated them to `!GH_ACTIONS_W22`. Did these tests fail on Windows 2016 and we just disabled them temporarily for Windows 2016?

Reply:
These tests only failed on windows 2016

Reply:
> @simpsont-oci should the unit tests failure be telling us the test that failed, or was this the script that failed: https://github.com/objectcomputing/OpenDDS/pull/3331/files#annotation_2790934180

This is the full output from the autobuild artifact... looks like the test script is just returning 1? Maybe it can't run the unit test executable?

```
#################### Test (auto_run_tests) [Thu Feb 17 21:51:51 2022 UTC]
Running: perl tests/auto_run_tests.pl   tests\static_ci_tests.lst -ExeSubDir Debug -Config STATIC_UNIT_TESTS -Config IPV6 -Config XERCES3 -Config CXX11 -Config RAPIDJSON in D:/a/OpenDDS/OpenDDS/OpenDDS
Test Lists: tests\static_ci_tests.lst
Configs: STATIC_UNIT_TESTS, IPV6, XERCES3, CXX11, RAPIDJSON, Win32, RTPS, GH_ACTIONS
Excludes: 

==============================================================================
auto_run_tests: tests/unit-tests/run_test.pl

.\UnitTests.EXE  -DCPSDebugLevel 1 -ORBVerboseLogging 1 -DCPSTransportDebugLevel 1 -ORBLogFile UnitTests.log -DCPSPendingTimeout 3 
UnitTests PID: 6160
2022-02-17 21:51:52: ERROR: UnitTests returned 3221225781 (started at 2022-02-17 21:51:52)
test FAILED.
auto_run_tests.pl: ERROR: "tests/unit-tests/run_test.pl" returned with status 1

auto_run_tests_finished: tests/unit-tests/run_test.pl Time:0s Result:1

#################### End [Thu Feb 17 21:51:52 2022 UTC]
```